cli/src/commands/sdk/remove/index.ts

```diff
@@ -148,10 +148,17 @@ export default class SDKRemoveCommand extends Command {
     this.log(chalk.red(" ERROR ") + chalk.dim(": " + message));
   }
 
-  private async confirmAction(message: string): Promise<boolean> {
+  private async confirmAction(message: string, defaultToContinue = true): Promise<boolean> {
     this.log(message);
-    const cont = ((await ask(chalk.dim(" -> "))) || "n").toLowerCase().trim();
+    const input = await ask(chalk.dim(" -> "));
+
+    if (input === "") {
+      clearLine(2);
+      return defaultToContinue;
+    }
+
+    const shouldContinue = (input || "n").toLowerCase().trim();
     clearLine(2);
-    return cont === "yes" || cont === "y";
+    return shouldContinue === "yes" || shouldContinue === "y";
```

Comment:
I think we can use inquirer for these.  Then we don't need `ask` or `clearLine` of our own